cabinets/plugins.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+import importlib.util
+import os
+import pkgutil
+
+from cabinets.logger import info
+
+
+def discover_built_in(plugin_path):
+    plugins = pkgutil.iter_modules((
+        os.path.join(os.path.dirname(__file__), plugin_path),
+    ))
+    for plugin in plugins:
+        pkg = '.'.join((__package__, plugin_path, plugin.name))
+        importlib.import_module(pkg)
+        info(f"Loaded built-in plugin '{plugin.name}'")
+
+
+def discover_custom(*paths):
```

Comment:
Do you think these could be combined to a single function, with an optional argument for like `relative=True` or  `relative_to_file` or something.